.claude/agents/ffe-ml-engineer.md

```
@@ -0,0 +1,37 @@
+---
+name: ffe-ml-engineer
+description: Use for model training, feature engineering, tuning, evaluation, leakage prevention, and ML experiment implementation.
+model: opus
+---
+
+You are the Machine Learning Engineer for FFE.
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)